### DISCUSSION TITLE
Getting the wrong dll version after install

### DISCUSSION THREAD

**Discussion by HudsonSchumaker** [msys2/MINGW-packages#11525] [Q&A]
Hello, 
i am using  mingw-w64-x86_64-toolchain and SDL2, SDL2_image has 
mingw-w64-x86_64-libjpeg-turbo dependence, when running the application the .jpg  file is not loaded. SDL2_image is looking for libjpeg-9.dll but seems on the [libjpeg](https://packages.msys2.org/package/mingw-w64-x86_64-libjpeg-turbo) the version is libjpeg-8.dll. How to update ?

Thanks in advance

**Comment by Biswa96** [comment]
Here is the dependency of SDL2_image.dll and there is no libjpeg-9.dll.

```
$ ntldd /ucrt64/bin/SDL2_image.dll
        KERNEL32.dll => C:\Windows\SYSTEM32\KERNEL32.dll (0x0000023f70f10000)
        libjpeg-8.dll => F:\msys64\ucrt64\bin\libjpeg-8.dll (0x0000023f70f10000)
        libpng16-16.dll => F:\msys64\ucrt64\bin\libpng16-16.dll (0x0000023f70f10000)
        SDL2.dll => F:\msys64\ucrt64\bin\SDL2.dll (0x0000023f70f10000)
        libtiff-5.dll => F:\msys64\ucrt64\bin\libtiff-5.dll (0x0000023f70f10000)
        libwebp-7.dll => F:\msys64\ucrt64\bin\libwebp-7.dll (0x0000023f70fa0000)
```

**Reply by HudsonSchumaker** [reply]
I can assure you that my build using this action [build win](https://github.com/SchumakerTeamStudios/bullet-hell-jam-2022/blob/main/.github/workflows/build_windows.yml) only works if i change the lib from 8 to 9
